docs/how-to-guides/model-inference-with-feathr.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Online Model Inference with Features from Feathr
+parent: How-to Guides
+---
+
+# Online Model Inference with Features from Feathr
+
+After you have materialized features in online store such as Redis or Azure Cosmos DB, usually end users want to consume those features in production environment for model inference.
+
+With Feathr's [online client](https://feathr.readthedocs.io/en/latest/#feathr.FeathrClient.get_online_features), it is quite straightforward to do that. The sample code is as below, where users only need to configure the online store endpoint (if using Redis), and call `client.get_online_features()` to get the features for a particular key.
+
+```python
+
+## put the section below into the initialization handler
+import os
+from feathr import FeathrClient
+
+# Set Redis endpoint
+os.environ['online_store__redis__host'] = "xx.redis.cache.windows.net"
```

Comment:
For _xx.redis.cache.windows.net_, user might not be aware this needs to be changed, change to something like <replace_with_your_redis_name>.redis.cache.windows.net?